### PR TITLE
Closes #46

### LINKED ISSUE #46: Notify administrator when new users sign up
As an administrator I wan't to know when new users sign in to the system so that I can chose to communicate with or upgrade a person immediately.

### PR DESCRIPTION
**Pull request by Timbozand**

### CODE REVIEW

File: src/main/java/se/lth/cs/connect/routes/Account.java [hunk 1]
```diff
@@ -107,10 +107,11 @@ protected void setup(PippoSettings conf) {
             String message = registerTemplate
                 .replace("{token}", token)
                 .replace("{hostname}", hostname);
-
+            
+            System.out.println(message); //REMOVE
```
Comment (smeets) [inline src/main/java/se/lth/cs/connect/routes/Account.java]: Yes

File: src/main/java/se/lth/cs/connect/routes/Account.java [hunk 2]
```diff
@@ -183,6 +185,11 @@ protected void setup(PippoSettings conf) {
             String email = AccountSystem.verifyEmail(token);
             if (email == null)
                 throw new RequestException("Invalid token: " + token);
+            
+            if(!frontend.equals("http://localhost:8181")){ //Dont wan't to send mail to serpconnect@cs.lth.se when we are testing.
```
Comment (smeets) [inline src/main/java/se/lth/cs/connect/routes/Account.java]: Remove this if -- always send email (see below for more)

File: src/main/java/se/lth/cs/connect/routes/Account.java [hunk 3]
```diff
@@ -183,6 +185,11 @@ protected void setup(PippoSettings conf) {
...
+            	app.getMailClient().
+    				sendEmail("serpconnect@cs.lth.se", "SERP connect email registration", "The following email has registered to SERP-connect:\n\n" + email);
```
Comment (smeets) [inline src/main/java/se/lth/cs/connect/routes/Account.java]: @emenlu do you read this email inbox?

Comment (emenlu) [reply src/main/java/se/lth/cs/connect/routes/Account.java]: Yes

Comment (smeets) [reply src/main/java/se/lth/cs/connect/routes/Account.java]: @Timbozand email addr and body needs to be configurable in the application.properties and messages_en.properties filef

**Review by smeets — CHANGES_REQUESTED** [review]

File: src/main/resources/conf/application.properties [hunk 4]
```diff
@@ -6,7 +6,7 @@ application.languages = en
 # Neo4j credentials
 neo4j.host = http://localhost:7474
 neo4j.username = neo4j
-neo4j.password = neo4j
+neo4j.password = hej
```
Comment (smeets) [inline src/main/resources/conf/application.properties]: This shit right here

**Review by smeets — CHANGES_REQUESTED** [review]

**Review by smeets — APPROVED** [review]
Ok, looks good!

**Comment by smeets** [comment]
[Notify administrator when new users sign up](https://trello.com/c/uFlPFWTN/65-notify-administrator-when-new-users-sign-up)

**Pull request merged**